presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestJdbcQueryBuilder.java

```diff
@@ -290,8 +290,8 @@ public void testBuildSqlWithDomainComplement()
             assertThat(lastQuery).isEqualTo("" +
```

Comment:
Do we have query in ATDQ to trigger this pushdown?

Reply:
yes, for example here
https://github.com/prestosql/presto/blob/8573d76f676b4535de6aaa29d3d739f114f3348b/presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueries.java#L1647